src/onefetch/cli.rs

```diff
@@ -65,6 +67,18 @@ impl Cli {
             .long("json")
                 .help("Outputs Onefetch in JSON format.")
             )
+        .arg(
+            Arg::with_name("yaml")
+            .short("y")
+            .long("yaml")
+                .help("Outputs Onefetch in YAML format.")
```

Comment:
Since this conflicts with the `--json` flag, I think that either
- [`conflicts_with("json")`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.conflicts_with) should be added (and also a `conflicts_with("yaml")` for the json `Arg`), or
- The `--json` and `--yaml` flags should be replaced with a `--format` `Arg` with the [possible values](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.possible_values) `json` and `yaml`

Personally, I think the latter would be better to keep the CLI as simple as possible 🙂

Reply:
I'm working on the latter, I'll be pushing a commit soon.